packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -152,6 +163,28 @@ export async function createCloudSQLInstance(celoEnv: string, instanceName: stri
   return [blockscoutDBUsername, blockscoutDBPassword, blockscoutDBConnectionName.trim()]
 }
 
+export async function createSecretInSecretManagerIfNotExists(
+  secretId: string,
+  secretLabels: string[],
+  secretValue: string
+) {
+  try {
+    await execCmd(`gcloud secrets describe ${secretId}`)
+
+    console.log(`Secret ${secretId} already exists, skipping creation...`)
+  } catch (error) {
```

Comment:
Not super clean, but I'd add a check for `helmdryrun` here to prevent creating a new secret when having that flag set.